src/modules/order/components/DiscountContainer/DiscountContainer.tsx

```diff
@@ -0,0 +1,38 @@
+/** @jsxImportSource @emotion/react */
+import { discounts } from 'data';
+import React, { useState } from 'react';
+import styles from './DiscountContainer.styles';
+
+interface DiscountContainerProps {
+  onDiscount: (percentage: number) => void;
+}
+
+export const DiscountContainer: React.FC<DiscountContainerProps> = (props) => {
```

Comment:
you can use destructuring here;
instead of (props) you can write `({onDiscount})`, and then you can call this function inside the component just by writing onDiscount

(now you have to write `props.onDiscount`)